README.md

```diff
@@ -22,9 +22,9 @@ You can find more examples [here.](https://github.com/joinfaces/joinfaces/wiki/E
 
 ## Key Features
 
-### Fifteen JoinFaces Starters available
```

Comment:
Maybe we should start using "many" instead of constantly changing readme's and wiki pages 🙈

Reply:
Ok.